src/Sia.Gateway/Requests/Playbook/GlobalActions/GetGlobalActions.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.Net.Http;
+using Sia.Domain.Playbook;
+using Sia.Gateway.Requests.Playbook;
+using Sia.Shared.Authentication;
+using Sia.Shared.Authentication.Http;
+using Sia.Shared.Requests;
+
+namespace Sia.Gateway.Requests
+{
+    public class GetGlobalActionsRequest : AuthenticatedRequest<IEnumerable<Action>>
+    {
+        public GetGlobalActionsRequest(AuthenticatedUserContext userContext)
```

Comment:
ctor seems redundant

Reply:
It doesn't compile without this constructor.